features/steps/recommendations_steps.py

```diff
@@ -0,0 +1,56 @@
+"""
+Recommendation Steps
+
+Steps file for Recommendation.feature
+
+For information on Waiting until elements are present in the HTML see:
+    https://selenium-python.readthedocs.io/waits.html
+"""
+from os import getenv
+import logging
+import json
+import requests
+from behave import *
+from compare import expect, ensure
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support.ui import Select
+from selenium.webdriver.support import expected_conditions
+
+WAIT_SECONDS = int(getenv('WAIT_SECONDS', '60'))
+
+@given('the following recommendations')
+def step_impl(context):
+    """ Delete all Recommendations and load new ones """
+    headers = {'Content-Type': 'application/json'}
+    #context.resp = requests.delete(context.base_url + '/recommendations/reset', headers=headers)
```

Comment:
commented this for now because RESET is not implemented. We'll need to add a new action for RESET to achieve this.